src/OrchardCore.Themes/TheAdmin/Views/Layout.cshtml

```diff
@@ -65,7 +65,7 @@
         <div id="ta-left-sidebar" class="d-flex flex-column justify-content-between align-items-stretch">
             @await RenderSectionAsync("Navigation", required: false)
             <div class="footer">
-                <button class="leftbar-compactor" type="button" title="@T["Collapse / expand menu"]"></button>
+                <a class="leftbar-compactor" title="@T["Collapse / expand menu"]"></a>
```

Comment:
This was fine as a `<button>`, just it shouldn't have the redundant `role` but have text.

Reply:
I wasn't sure what to put between the buttons as a text. What should I do?

Reply:
Just the same: `@T["Collapse / expand menu"]`.

Reply:
Are you sure? The button looks like this:
![image](https://user-images.githubusercontent.com/79921246/159792842-ad4af61b-7081-4fae-9801-b1a1ceb77544.png)

I don't think we just add a simple text there.
![image](https://user-images.githubusercontent.com/79921246/159793001-a9d938a7-7e0a-4411-a3ac-ce07508cee9e.png)

Reply:
I'm sure (this is how a `<button>` works) but you have to adjust the CSS, of course.

Reply:
What should I do with the CSS? Should I hide the text? Or should I make the text fit there, even if the text wasn't there before?

Reply:
Hide it. Screen readers should be able to process it nevertheless.